tools/BcosAirBuilder/build_chain.sh

```diff
@@ -1922,19 +1924,20 @@ deploy_nodes()
     generate_chain_cert "${sm_mode}" "${ca_dir}"
 
     for line in ${ip_array[*]}; do
-        if [ -n "$(echo ${line} | grep "\.")" ];then
-            ip=${line%:*}
-            num=${line#*:}
-        else
+        if echo "$line" | grep -E "\[.*\]:[0-9]+" >/dev/null; then
             ip=${line%\]*}
             ip=${ip#*\[}
             num=${line#*\]}
             num=${num#*:}
             use_ipv6="true"
+            if [[ -n "${use_ipv6}" && -z "$(echo "$ip" | grep -E "^([0-9a-fA-F]{0,4}:){1,7}[0-9a-fA-F]{1,4}$")" ]]; then
+            LOG_FATAL "Please check IPv6 address: ${ip}."
+            fi
+        else
+            ip=${line%:*}
+            num=${line#*:}
         fi
-        if [[ -n "${use_ipv6}" && -z "$(echo "$ip" | grep -E "^([0-9a-fA-F]{0,4}:){1,7}[0-9a-fA-F]{1,4}$")" ]]; then
-            LOG_WARN "Please check IPv6 address: ${ip}, if you use domain name please ignore this."
-        elif [[ -z "${use_ipv6}" && -z $(echo $ip | grep -E "^[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}$") ]]; then
```

Comment:
why remoce ipv6 check? Is it possible keep it?

Reply:
we move this check to if branch, so we don't need to keep it